### PR TITLE
chore(ci): extract package signing from release-draft MONGOSH-1746

### PR DESCRIPTION
**Pull request by addaleax**

### CODE REVIEW

File: .evergreen/sign-packaged-artifact.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+export NODE_JS_VERSION=${NODE_JS_VERSION}
+export ARTIFACT_URL_FILE="$PWD/../artifact-url.txt"
+
+# TODO: Remove this once it is confirmed that it's safge to do so
```
Comment (mabaasit) [inline .evergreen/sign-packaged-artifact.sh:8]: do we need to link the ticket here?

Comment (mabaasit) [reply .evergreen/sign-packaged-artifact.sh:8]: sorry, but do we even need this as we are using garasign? (or is this particular build a special one?)

File: packages/build/src/index.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ if (require.main === module) {
         'upload',
         'draft',
         'publish',
+        'sign',
         'trigger-release',
       ].includes(command)
     ) {
```
Comment (mabaasit) [inline packages/build/src/index.ts]: Do we need to add `sign` to the error below as well?

**Review by mabaasit — APPROVED** [review]

File: packages/build/src/index.ts (hunk 2 above)
Comment (mabaasit) [reply packages/build/src/index.ts]: looked at the next commit and its taken care of :)

File: .evergreen/sign-packaged-artifact.sh (hunk 1 above)
Comment (addaleax) [reply .evergreen/sign-packaged-artifact.sh]: Yeah, thank you for the reminder! I think we can just remove this, that's exactly what I was hoping for, just forgot about this TODO myself 😅

**Pull request merged**